requirements.txt

```diff
@@ -1,3 +1,3 @@
-pyjwt==1.6.0
-blinker==1.4
+pyjwt>=1.6.0
+blinker>=1.4
```

Comment:
Why did you change this?

Reply:
To align it with the values inside _setup.py_ -- is this the wrong way to do it? The _setup.py_ had "pyjwt>=1.0.0", but _requirements.txt_ had a different "pyjwt==1.6.0" So for consistency, made both reference the same v1.6.0 and made them both ">=". For consistency, also added a ">=" for _blinker_, picking the most recent release.

Reply:
The Python Packaging Authority [user guide](https://packaging.python.org/discussions/install-requires-vs-requirements/) and this article on [setup.py vs requirements.txt](https://caremad.io/posts/2013/07/setup-vs-requirement/) says the current best practice is to have ">=" in the _setup.py_ and for the _requirements.txt_ to reference _setup.py_ or only have "==". But that makes an assumption on how the recipient intends to use those files. Maybe there should only be a _setup.py_ and no _requirements.txt_ file?

Fixing this sounds like a new issue in itself. Perhaps accept the changes that are in this pull request since it is better than it was (having them consistent is better than having them inconsistent), and a new issue should be opened to determine an even better fix.

Reply:
Opened issue https://github.com/oauthlib/oauthlib/issues/724 and resolving this conversation.